extensions/patterns/configure_ec2/exec_env/execution-environment.yml

```diff
@@ -0,0 +1,29 @@
+---
+version: 3
+
+dependencies:
+  ansible_core:
+    package_pip: ansible-core
+  ansible_runner:
+    package_pip: ansible-runner
+  galaxy:
+    collections:
+      - name: cloud.aws_ops
+        source: https://github.com/redhat-cop/cloud.aws_ops.git
+        type: git
+        version: main
+      - name: amazon.aws
+        source: https://github.com/ansible-collections/amazon.aws.git
+        type: git
+        version: main
+  python:
+    - boto3
+    - botocore
```

Comment:
Should we specify any boto3 and botocore requirements here? The minimum supported version at least.

Reply:
This will install the latest versions into the EE. Is there any reason not to do that?